src/accelerate/components/ListItem.tsx

```diff
@@ -131,7 +135,13 @@ const ListItem = function ( props: {
 						</div>
 					</div>
 					<div className='record-name__title'>
-						<a href={ post.url || post.editUrl || '' } onClick={ () => trackEvent( listId, 'Navigate', { type: post.type } ) }>
+						<a href={ post.url || post.editUrl || '' } onClick={ ( e: React.MouseEvent ) => {
+							if ( onManage ) {
+								onEdit( e );
```

Comment:
Given this change means there's no need to check for `onManage` in the `onEdit` function above it seems to me the entire `onEdit` function might as well be what's passed in for the `onManage` prop. That would remove code from this function that may never be used.

Reply:
`onEdit()` is still used on links to edit content, even if `onManage()` is not defined/passed. And we still need to do the check either way because it's an optional callback.

Reply:
Aha, I missed the call to `onEdit` further down 👍